public/build/css/all.css

```diff
@@ -1,3 +1,7 @@
+/*!
```

Comment:
When @joecohens compiles this file, you always see this header disappear, but when you compile it, it comes back? We're not using font awesome. Why is it still here?